applications/nrf_desktop/description.rst

```diff
@@ -357,7 +357,7 @@ Depending on the development kit you use, you need to select the respective conf
 
       .. table-from-rows:: /includes/sample_board_rows.txt
          :header: heading
-         :rows: nrf52840dongle_nrf52840, nrf52833dongle_nrf52833, nrf52820dongle_nrf52820
+         :rows: nrf52840dongle_nrf52840_bare, nrf52833dongle_nrf52833, nrf52820dongle_nrf52820
```

Comment:
The "includes/sample_board_rows.txt" will also require an update. This will fix the doc build error.